sdk/core/azure-core/tests/async_tests/test_basic_transport_async.py

```diff
@@ -1038,3 +1047,43 @@ async def test_close_too_soon_works_fine(caplog, port, http_request):
     result = await transport.send(request)
 
     assert result  # No exception is good enough here
+
+
+@pytest.mark.skipif(
+    parse_version(aiohttp.__version__) >= parse_version("3.10"),
+    reason="aiohttp 3.10 introduced separate connection timeout",
+)
+@pytest.mark.parametrize("http_request", HTTP_REQUESTS)
+@pytest.mark.asyncio
+async def test_aiohttp_timeout_response(http_request):
```

Comment:
One of these tests targets aiohttp <3.10 (previous behaviour), and one >=3.10 (new behaviour).
My hope is that our min-requirements test passes will mean that both of these get exercised in different environments, but if not, then it looks like in dev_requirements.txt, the pypy test pass will explicitly use aiohttp 3.8, so both tests should get run at least once.